### ISSUE TITLE
IE11 Angular 2.1.2 -> 2.2.0 , new problems

### ISSUE THREAD

**Issue by montella1507** [angular/angular#13302]
Hi,
is there any working Angular 2 plunker with latest version of angular that works in IE11?

I have some strange behavior that i want to show and help with fix. Unfortunately it is in big non-public project, so i want to prepare minimal reproduce plunks.. 

Is there any base plunk that i can (or anyone else) fork? 

The problem is, evething is working right in @angular 2.1.2 in IE11 but in version 2.2.0 there are big issues in IE11 without any change in code - only different version of Angular. 

There is no error in Console in IE11, even the clear page with Reactive Forms doesnt work, the application just keep taking more and more memory. 

I would like to help and simulate it in plunker but i am not able to prepare Plunk from start. So if anyone could help me with base plunks  i can prepare plunks with reproducing the problems.

**Comment by marcovdb** [comment]
Yeah, I ran into the same issue. The Plunkr doesn't work in IE11.

For what it's worth, this is the error message I'm getting:

```
Error: (SystemJS) TypeError: Unable to get property 'apply' of undefined or null reference
	   at combine (eval code:2800:9)
	   at Anonymous function (eval code:2648:5)
	   at ZoneDelegate.prototype.invoke (https://unpkg.com/zone.js/dist/zone.js:227:13)
	   at Zone.prototype.run (https://unpkg.com/zone.js/dist/zone.js:113:17)
	Error loading http://run.plnkr.co/y5gXhtvC3JpzhsEA/src/main.ts
   {
      [functions]: ,
      __proto__: { },
      description: "(SystemJS) TypeError: Unable to get property 'apply' of undefined or null reference
	   at combine (eval code:2800:9)
	   at Anonymous function (eval code:2648:5)
	   at ZoneDelegate.prototype.invoke (https://unpkg.com/zone.js/dist/zone.js:227:13)
	   at Zone.prototype.run (https://unpkg.com/zone.js/dist/zone.js:113:17)
	Error loading http://run.plnkr.co/y5gXhtvC3JpzhsEA/src/main.ts",
      message: "(SystemJS) TypeError: Unable to get property 'apply' of undefined or null reference
	   at combine (eval code:2800:9)
	   at Anonymous function (eval code:2648:5)
	   at ZoneDelegate.prototype.invoke (https://unpkg.com/zone.js/dist/zone.js:227:13)
	   at Zone.prototype.run (https://unpkg.com/zone.js/dist/zone.js:113:17)
	Error loading http://run.plnkr.co/y5gXhtvC3JpzhsEA/src/main.ts",
      name: "Error",
      originalErr: { },
      originalStack: undefined,
      stack: null
   }
``

**Comment by DzmitryShylovich** [comment]
Duplicate of #13152

**Comment by montella1507** [comment]
Not guys. this has nothing to do with plunker. As i have told you. I have problems In my own solutions. Not in the plunker. It **is BIG application that works like a charm with angular 2.1.2. 

But after version 2.2.0 it doesnt work without any error in the console.**  So do you think, that the additional SHim is required only AFTER version 2.2.0? Because i have tried to add it to polyfills.ts and it didnt work anyway. The same  fail was when i added it directly to HTML.

@DzmitryShylovich  despite of our argueing, we want the same thing, to be Angular 2 successfull. So i can show you my project (give you the link to 2 same solutions, only with different angular verison )and you will see the application work. Or if you provide "base plunker" with the latest Angular 2 version i will try to prepare minimal repro.

**Comment by DzmitryShylovich** [comment]
it's a known issue https://github.com/angular/angular/issues/13152
there's no point in reporting it again and again. it doesn't change anything. Just subscribe to https://github.com/angular/angular/issues/13152 updates

**Comment by zoechi** [comment]
@montella1507 just because the other issue mentions that even the default Plunker template doesn't run in IE doesn't mean it's a Plunker-related issue. It's a known issue with IE in general that can be reproduced even with the default Plunker template.

**Comment by montella1507** [comment]
OK, thank you. Then i am m sorry for taking your time, my bad. I will close this.
